extension/src/content.js

```diff
@@ -52,7 +52,13 @@ async function init() {
 					mutation.addedNodes && mutation.addedNodes.forEach((node) => {
 						// Check className each time as mutations seem to be live
 						// Condition: Is the entry form and has translation fields so we want to show an export button
-						if(node.className && node.className.indexOf && node.className.indexOf('editCreateEntryWrapper') !== -1 && node.className.indexOf('export-button-added') === -1 && node.querySelector('div.translationField') !== null) {
+						if ((isLegacy() && node.className && node.className.indexOf && node.className.indexOf('editCreateEntryWrapper') !== -1
+							&& node.className.indexOf('export-button-added') === -1
+							&& node.querySelector('div.translationField') !== null)
+							|| (!isLegacy() && node.parentElement && node.parentElement.dataset && node.parentElement.dataset.test === 'LevelComponent'
+							&& node.className.indexOf('export-button-added') === -1
+							&& [...node.querySelectorAll("*")].filter(n => n.name && (n.name.endsWith('EN') || n.name.endsWith('RU') || n.name.endsWith('PTBR') || n.name.endsWith('ESLA'))).length > 0
```

Comment:
What's the reasoning for supporting only certain languages?

Reply:
GraphCMS does not provide a "translations" collection anymore and the list of available locales for a certain project needs to be fetched from somewhere else, and there is no other marker on the fields to indicate they are translated fields.

So, to get it to work specifically for GreySnow's case, I've limited it for these locales, but I have already found where I can get the list of locales for the project and will be making another change soon to makes sure it works for any locales available.

Reply:
Okay, I'll merge and await for the next change before pushing a release. I doubt anybody uses the extension but I wouldn't want to break it for them if they do.